advanced_source/dispatcher.rst

```diff
@@ -105,6 +105,50 @@ speaking, the structure of your registrations will look like this:
     that provides implementations for all basic operators on the XLA dispatch
     key.
 
+
+For operators that do not need autograd
+---------------------------------------
+
+In the next section, we will discuss how to add autograd support to an operator.
+But for the ops that do not need autograd support, the following line can be
+added to improve useability and make your op behave like PyTorch's built-in
+operators.
+
+.. code-block:: cpp
+
+  REGISTER_AUTOGRAD_NOT_IMPLEMENTED_FALLBACK(myops, "myadd");
```

Comment:
I'm not a big fan of the macro API here. (1) It's not namespaced with TORCH, so it is potentially ambiguous with other macros, and (2) it's pretty short to do this without a macro, right? So it seems better to not do the macro and make people write it out.

Reply:
I also still think that we ought to be registering a global fallback for all operators that defaults us to assuming that autograd is not implemented...

Reply:
> it's pretty short to do this without a macro, right? So it seems better to not do the macro and make people write it out.

It will be like 6 lines, so not too bad. I'd be okay with that. Alternatively, we'd prefix this macro with `TORCH_`.

> I also still think that we ought to be registering a global fallback for all operators that defaults us to assuming that autograd is not implemented

Yeah this seems reasonable. Creating an issue to discuss the implications of this in more detail...

Also, since this has been merged (accidentally), I will make the changes to the tutorial in a separate PR